comp/trace/bundle_test.go

```diff
@@ -12,26 +12,25 @@ import (
 	"github.com/stretchr/testify/require"
 	"go.uber.org/fx"
 
-	coreconfig "github.com/DataDog/datadog-agent/comp/core/config"
-	"github.com/DataDog/datadog-agent/comp/trace/config"
+	"github.com/DataDog/datadog-agent/comp/core/config"
 	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
 )
 
 // team: agent-apm
 
+// TODO(AIT-8301): Why are these tests green if not all elements are supplied?
```

Comment:
You have this issue because you have to explicitly instance your main components: https://github.com/DataDog/datadog-agent/pull/19466/files#diff-f25a1b5a7f28bd1a8890759fff9c2b2e0d0b8bbfc3ab1ba70bc4eb5aad6ef8bfL24-L27.